check-environment-and-install

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+[ "${BASH_SOURCE[0]}" != "$0" ] && { echo "ERROR: this script must be executed by Bash, neither source'd nor executed by a different shell"; return 1; }
+
+set -e
+if [[ -n "${BROWSER_DRIVER_INSTALLER_CHROME_VERSION:-}" && -n "${BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH:-}" ]]; then
```

Comment:
add a blank line after set -e

we probably want set -u as well (undefined vars)

Reply:
added blank line, adding `set -u` would render the error messages useless

Reply:
how so?